docs/how_to_contribute.md

```diff
@@ -160,6 +160,125 @@ Here is an example of these rules applied:
         pass
 
 
+## Testing your code
+
+Pinax apps typically support several different versions of Python and several
+different version of Django. The supported combination are specified in `tox.ini`
```

Comment:
Typo: ."..and several different version**s** of Django". Also: "The supported combination**s**..."